src/api/configuration.cpp

```diff
@@ -46,42 +46,9 @@ namespace mamba
             {
                 if (value.empty() || (value == "true") || (value == "1") || (value == "<true>"))
                 {
-                    if (on_linux)
-                    {
-                        std::array<std::string, 6> cert_locations{
-                            "/etc/ssl/certs/ca-certificates.crt",  // Debian/Ubuntu/Gentoo etc.
-                            "/etc/pki/tls/certs/ca-bundle.crt",    // Fedora/RHEL 6
-                            "/etc/ssl/ca-bundle.pem",              // OpenSUSE
-                            "/etc/pki/tls/cacert.pem",             // OpenELEC
-                            "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem",  // CentOS/RHEL 7
-                            "/etc/ssl/cert.pem",                                  // Alpine Linux
-                        };
-                        bool found = false;
-
-                        for (const auto& loc : cert_locations)
-                        {
-                            if (fs::exists(loc))
-                            {
-                                value = loc;
-                                found = true;
-                            }
-                        }
-                        if (!found)
-                        {
-                            LOG_ERROR << "ssl_verify is enabled but no ca certificates found";
-                            throw std::runtime_error("No CA certificates found.");
-                        }
-                    }
-                    else
-                    {
-                        value = "<system>";
-                    }
+                    value = "<system>";
```

Comment:
will this value contain `<system>` for rhumba now? in that case, that would be the wrong value (for non-statically compiled rhumba).

Reply:
I just moved the find operation to init_curl_ssl, and also added back the pre-processor guard UMAMBA_STATIC that prevent setting SSL backends into that same function